resources/codegen/tests/test___main__.py

```diff
@@ -213,8 +213,10 @@ def test_when_none_set_then_all_languages_are_generated(self, tmp_path, yaml_pat
 
 
 def execute_cli(*args):
-    all_args = [sys.executable, "-m", "openassetio_codegen"]
-    all_args.extend(args)
+    # Str wrapping prevents issues with Path objects in Windows/3.7
```

Comment:
[Consider][Minor] Windows/3.7 could be a tad confusing. Maybe Windows/Python 3.7?

Reply:
Updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/694/commits/819f332c17bfafb11dd74261254121a439f03cf1).